### PR TITLE
Update to include BNB for SDAO

### PR DESCRIPTION
**Pull request by RobertSDAO**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/sdao/index.js exports TVL: 
        
 
 
    
    total                    40.81 M

File: package-lock.json [hunk 1]
```diff
@@ -20347,7 +20347,7 @@
       "requires": {
         "base64-url": "^2.3.3",
         "eventsource": "^1.1.0",
-        "node-fetch": "2.6.7",
+        "node-fetch": "^2.6.1",
```
Comment (g1nt0ki) [inline package-lock.json:20350]: also, can you undo the change here

File: projects/sdao/index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const { default: axios } = require("axios");
```
Comment (g1nt0ki) [inline projects/sdao/index.js:1]: this file is no longer used, can you delete it please?

File: pull_request_template.md [hunk 3]
```diff
@@ -1,40 +1,27 @@
 ##### Twitter Link:
```
Comment (g1nt0ki) [inline pull_request_template.md:1]: can you undo the changes to this file

**Review by g1nt0ki — CHANGES_REQUESTED** [review]

**Comment by llamatester** [comment]
The adapter at projects/singularityDAO/index.js exports TVL: 
        
 
 
    ethereum                  2.04 M
    ethereum-staking          26.85 M
    bsc-pool2                 684.30 k
    bsc-staking               7.34 M
    ethereum-pool2            1.79 M
    staking                   34.18 M
    pool2                     2.48 M
    
    total                    2.04 M

**Comment by llamatester** [comment]
The adapter at projects/singularityDAO/index.js exports TVL: 
        
 
 
    ethereum-staking          26.37 M
    bsc-staking               7.19 M
    ethereum                  2.03 M
    bsc-pool2                 678.56 k
    ethereum-pool2            1.78 M
    staking                   33.55 M
    pool2                     2.46 M
    
    total                    2.03 M
